website_and_docs/content/blog/2023/whats-new-in-selenium-manager-with-selenium-4.11.0.md

```diff
@@ -0,0 +1,125 @@
+
+One of the key differences between a regular Chrome release and CfT is that Chrome is *evergreen*, i.e., it upgrades automatically and silently to the next stable version when available. On the other hand, CfT is not *evergreen*, to allow pined browsers for testing. CfT releases are portable binaries (for Windows, Linux, and macOS) for different versions, including the stable, beta, dev, and canary channels. These releases can be programmatically discovered using the [CfT JSON endpoints](https://github.com/GoogleChromeLabs/chrome-for-testing#json-api-endpoints). 
+
+As of version 114, the chromedriver team has stopped publishing the chromedriver releases and metadata using their traditional [chromedriver download repository](https://chromedriver.chromium.org/downloads). This way, and as of version 115, chromedriver releases can only be discovered programmatically using the abovementioned [CfT JSON endpoints](https://github.com/GoogleChromeLabs/chrome-for-testing#json-api-endpoints).
+
+This change has been very relevant for the so-called driver managers. Luckily, Selenium already supports this new way of chromedriver discovery. The last version of Selenium Manager uses the CfT endpoints for chromedriver management. Therefore, if you are using Selenium Manager and Chrome, you must be updated to Selenium 4.11.0 to continue managing chromedriver automatically.
+
+### Automated browser management
+
+Moreover, as of Selenium 4.11.0, Selenium Manager implements automated browser management based on CfT. In other words, Selenium Manager uses the CfT endpoints to discover, download, and cache the different CfT releases, making them seamlessly available for Selenium. Let's consider the following line of code (this is Java, although the behavior is the same in all the Selenium binding languages):
+
+```
+WebDriver driver = new ChromeDriver();
+```
+
+So far, Selenium Manager manages chromedriver for us. This way, all the complexity related to CfT endpoints, driver versions, etc., is transparent, and we can rely on Selenium Manager to discover, download, and cache the proper driver for us.
+
+In addition, and as a significant novelty starting on Selenium 4.11.0, if Chrome is not installed on the local machine when executing the previous line, the current stable CfT release is discovered, downloaded, and cached (in `~/.cache/selenium/chrome`). But there is more. In addition to the stable CfT version, Selenium Manager also allows downloading of older versions of CfT (starting in version 113, which is the first version published as CfT). 
+
+Let's consider a simple example. The following lines will download CfT 114 and its proper chromedriver version. Then Selenium will start Chrome to be driven programmatically, as usual:
```

Comment:
This isn't as clear. If installed version is 114 it uses that and if it isn't it downloads it for you?

Reply:
Done (rephrased to clarify).